components/scrubber/scrubber.go

```diff
@@ -94,11 +106,22 @@ func (s *scrubberImpl) JSON(msg json.RawMessage) (json.RawMessage, error) {
 
 // KeyValue implements Scrubber
 func (s *scrubberImpl) KeyValue(key string, value string) (sanitisedValue string) {
-	f, ok := s.FieldIndex[strings.ToLower(key)]
-	if !ok {
-		return value
+	return s.GetSanitisatiser(key)(value)
+}
+
+// GetSanitisatiser implements
+func (s *scrubberImpl) GetSanitisatiser(key string) Sanitisatiser {
+	if f, ok := s.FieldIndex[key]; ok {
+		return f
+	}
+	for regexp, f := range s.RegexpIndex {
+		if regexp.MatchString(key) {
```

Comment:
it allows to cover more properties, i.e. token will cover as well `owner_token`, and `contexturl` covers such common fields like `workspaceContextURL` or `wsContextUrl` without a need to list each